### ISSUE TITLE
Errors with functions that do not have "return" keyword

### ISSUE THREAD

**Issue by LalehB** [IntelLabs/ParallelAccelerator.jl#20]
For the cases that a function doesn't have a return keyword, there is an error as follows:

```jl
/var/folders/vq/rxp7r5yj3dj7mn1mtmmsc8nm0000gn/T/tmpCkFp7y/cgen_output0.cpp:187:8: error: use of undeclared identifier 'nothing'
return nothing;
       ^
1 error generated.
ERROR: LoadError: failed process: Process(`g++ -O3 -std=c++11 -g -fpic -c -o /var/folders/vq/rxp7r5yj3dj7mn1mtmmsc8nm0000gn/T/tmpCkFp7y/cgen_output0.o /var/folders/vq/rxp7r5yj3dj7mn1mtmmsc8nm0000gn/T/tmpCkFp7y/cgen_output0.cpp`, ProcessExited(1)) [1]
```

for a code example as follow: 

```jl
@acc function UpdateCenter!(X,label,centroids,numCenter,numPoint,dim)
	count = Array(Int64,numCenter)
	for j= 1:numPoint
		centroids[label[j]] = centroids[label[j]] .+ X[j]
		count[label[j]] +=1;
	end
	for f = 1:numCenter
		for d = 1:dim
			centroids[f,d] /= count[label[f]]
		end
	end

end
```

Which does not have a return.

**Comment by LalehB** [comment]
added the highlight

**Comment by lkuper** [comment]
@LalehB, thanks for filing this.

@DrTodd13, @ninegua, @ehsantn -- do accelerated functions have to return something?  Would it be easy to drop this requirement?  If not, we should at least mention it in the README.  (The issue seems to be whether the function returns something and not the presence or absence of the `return` keyword as such.)

**Comment by DrTodd13** [comment]
Every Julia function returns something so it isn't that it doesn't have a return keyword, it is just that the last statement in the function returned the value "nothing." This should be a pretty easy fix once I get around to it. 

Todd 

----- Original Message -----

**Comment by lkuper** [comment]
Right -- by "returns something" I mean "returns a non-nothing value".

**Comment by DrTodd13** [comment]
Fixed by 09035ba3cc7d601d56a0f3821c623408a204381c.
